src/redux/migrations.test.ts

```diff
@@ -571,4 +572,14 @@ describe('Redux persist migrations', () => {
 
     expect(migratedSchema).toMatchObject(expectedSchema)
   })
+
+  it('works for v56 to v57', () => {
+    const oldSchema = v56Schema
+    const migratedSchema = migrations[57](oldSchema)
+
+    const expectedSchema: any = _.cloneDeep(oldSchema)
+    expectedSchema.app.visualizeNFTsEnabledInHomeAssetsPage = false
```

Comment:
thanks for being thorough! this test is obviously fine, but going forward we only need to write tests for more complex migrations where maybe variables are moving around or we derive new state from previous state. it's fine to have this but you can save some effort next time for a migration that adds one variable :)